ironfish/src/rpc/routes/wallet/exportAccount.ts

```diff
@@ -9,7 +9,7 @@ export type ExportAccountRequest = { account?: string }
 export type ExportAccountResponse = {
   account: {
     name: string
```

Comment:
This is no longer a required value. We will need to handle this with the `export` command updates, included in spec.